### ISSUE TITLE
[Bug]: Override msg value

### ISSUE THREAD

**Issue by RabahZeineddine** [GogoVega/node-red-contrib-firebase-realtime-database#31]
### What happened?

When using firebase-get it overrides the msg content, like msg.req, msg.res and the flow can't finish the request.

### How to reproduce?

1. Create an http in node
2. add firebase-get node to fetch data from db
3. add http response on the firebase-get output.

### Additional comments?

Additional comments here, if any.

**Comment by GogoVega** [comment]
Hi Rabah,
The current version (0.3.1) of the `Firebase get` node does not support the option to pass through the message.
This is why your `http request` node cannot complete its request because the message entering the `Firebase get` node is not passed to its output.

This feature was developed [here](#29) and will be released for v0.4.0 before the end of the weekend.

If you need a temporary solution, I can suggest you to use context:
```js
// Before the Firebase get Node
flow.set('request', msg);

// After the Firebase get Node
const msg2passThrough = flow.get('request') || {};

msg = {
  ...msg2passThrough,
  payload: msg.payload,
};

return msg;
```

**Comment by RabahZeineddine** [comment]
> Hi Rabah, The current version (0.3.1) of the `Firebase get` node does not support the option to pass through the message. This is why your `http request` node cannot complete its request because the message entering the `Firebase get` node is not passed to its output.
> 
> This feature was developed [here](https://github.com/GogoVega/node-red-contrib-firebase-realtime-database/pull/29) and will be released for v0.4.0 before the end of the weekend.
> 
> If you need a temporary solution, I can suggest you to use context:
> 
> ```js
> // Before the Firebase get Node
> flow.set('request', msg);
> 
> // After the Firebase get Node
> const msg2passThrough = flow.get('request') || {};
> 
> msg = {
>   ...msg2passThrough,
>   payload: msg.payload,
> };
> 
> return msg;
> ```

Hi @GogoVega , 

Thank you for your reply. 

That's what I did, I used the change node to use the context. 

![image](https://user-images.githubusercontent.com/19435288/229137632-42d3d425-50fa-4912-b7a2-3d20e2c04058.png)
